.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    # We use a standard Debian image to mirror a typical developer environment.
+    # This should be updated whenever a new Debian stable version is available.
```

Comment:
I don't think this is correct since bookworm is available, it's mostly when we move SDW over to the new version. I don't think we need to annotate our choice, we haven't in most other CI manifests.

Reply:
Fair enough; we have the same comment in the docs repo (https://github.com/freedomofpress/securedrop-docs/blob/main/.github/workflows/ci.yml) so probably should remove it there as well.